cmd/restore-operator/README.md

```diff
@@ -0,0 +1,133 @@
+etcd-restore-operator   10.3.243.216   <none>        19999/TCP   51s
+```
+
+The etcd-restore-operator will also create the `EtcdRestore` CRD:
+
+```sh
+$ kubectl get crd
+NAME                                       KIND
+etcdclusters.etcd.database.coreos.com      CustomResourceDefinition.v1beta1.apiextensions.k8s.io
+etcdrestores.etcd.database.coreos.com      CustomResourceDefinition.v1beta1.apiextensions.k8s.io
+```
+
+### Create the AWS secret
+
+Every restore request will specify a secret containing AWS credentials and config that will be used by the etcd-restore-operator to access S3 for retrieving the backup snapshot.
+
+Verify that the local aws config and credentials files exist:
+
+```sh
+$ cat $AWS_DIR/credentials
+[default]
+aws_access_key_id = XXX
+aws_secret_access_key = XXX
+
+$ cat $AWS_DIR/config
+[default]
+region = <region>
+```
+
+Create the Kubernetes secret `aws`:
+
+```sh
+kubectl create secret generic aws --from-file=$AWS_DIR/credentials --from-file=$AWS_DIR/config
+```
+
+### Create EtcdRestore CR
+
+An `EtcdRestore` CR specifies the following parameters of a restore request:
+- `metadata.name`: the CR name specifies the name of the newly restored cluster
+- `spec.clusterSpec`: the new cluster spec
+- `spec.s3`: the S3 restore source specifying the aws secret name and the path to the backup snapshot 
+
+Fill the template [example/etcd-restore-operator/restore_cr.yaml][restore-cr] with concrete `path` and `awsSecret` values and create the CR to trigger a restore.
+
+**Note:** The `path` must be of the format `<s3-bucket-name>/<path-to-backup>`.
+
+```sh
+sed -e 's|<full-s3-path>|mybucket/etcd-snapshot.backup|g' \
+    -e 's|<aws-secret>|aws|g' \
+    example/etcd-restore-operator/restore_cr.yaml \
+    | kubectl create -f -
+```
+
+### Verify the restore status and restored cluster:
+
+Check the `status` section of the `EtcdRestore` CR:
+
+```sh
+$ kubectl get etcdrestore restored-etcd-cluster -o yaml
+apiVersion: etcd.database.coreos.com/v1beta2
+kind: EtcdRestore
+...
+status:
+  succeeded: true
+```
+
+On success the etcd-restore-operator will have created an `EtcdCluster` CR and the etcd-operator will have scaled the cluster to the desired size.
+```
+$ kubectl get etcdcluster
+NAME                    KIND
+restored-etcd-cluster   EtcdCluster.v1beta2.etcd.database.coreos.com
+
+$ kubectl get pods
+NAME                                     READY     STATUS    RESTARTS   AGE
+etcd-operator-2486363115-ltc17           1/1       Running   0          1h
+etcd-restore-operator-4203122180-npn3g   1/1       Running   0          30m
+restored-etcd-cluster-0000               1/1       Running   0          8m
+restored-etcd-cluster-0001               1/1       Running   0          8m
+restored-etcd-cluster-0002               1/1       Running   0          8m
+```
+
+### Cleanup
+
+Delete the etcd-restore-operator deployment and service, and the `EtcdRestore` CR. Be careful not to delete the newly created `EtcdCluster` CR.
```

Comment:
`deployment and service, and the EtcdRestore CR` -> `deployment, service, and the EtcdRestore CR.`?

Reply:
"Be careful not to delete the newly created `EtcdCluster` CR."
SInce this is a demo, user might want to delete the newly created etcd cluster for cleanup right?

maybe say:
"Delete the newly created `EtcdCluster` CR to delete the restored etcd cluster"?

Reply:
Changing it to: "Note that deleting the `EtcdRestore` CR won't delete the `EtcdCluster` CR."
Just want to let the user know that fact.

Reply:
sounds good!